src/test/test_faceter.cc

```diff
@@ -57,23 +54,23 @@ TEST_CASE("Faceting BREP and writing to MOAB", "[faceter]") {
   double xyz[3] = {1.0, 1.0, 1.0};
   ret = gqt.point_in_volume(vol1, xyz, result);
   REQUIRE(ret == moab::MB_SUCCESS);
-  REQUIRE(result == 1);
+  CHECK(result == 1);
```

Comment:
I would ideally have these changes from REQUIRE to CHECK in a separate commit, but I think it doesn't matter.